src/main/java/io/mbab/sda/groupproject2/menu/action/CreateSongAction.java

```diff
@@ -42,7 +41,7 @@ public void execute() {
     var album = builder.album(input).build();
     System.out.println("Podaj długość utworu ");
 
-    var input2 = scanner.number();
+    var input2 = scanner.nextDouble();
```

Comment:
input2 t nie jest dobra nazwa, moze lepiej to co wisujemy czyli 
double length = scanner.nextDouble();